cmd/kinko/main.go

```diff
@@ -426,6 +428,24 @@ func secretsFromCLIFlags() (map[string][]byte, error) {
 		}
 		secrets[v] = secret
 	}
+
+	for _, v := range BlobSecrets {
+		substrings := strings.Split(v, "=")
```

Comment:
Hi @ken8203,

Thanks. I think it would be better to use `SplitN` and limit it to cut 2 chunks.

```suggestion
		substrings := strings.SplitN(v, "=", 2)
```